druntime/posix.mak

```diff
@@ -371,6 +374,14 @@ $(ROOT)/valgrind.o : src/etc/valgrind/valgrind.c src/etc/valgrind/valgrind.h src
 	@mkdir -p `dirname $@`
 	$(CC) -c $(CFLAGS) $< -o$@
 
+################################################
+
+gen_tagged_srcs:
+	./cp_tagged_hier.sh ./impl $(TGEN_DIR) $(ARCH),$(OS)
```

Comment:
Putting all arch/platform-specific sources into a directory named `config` is more common than `impl` - at least in the C/C++ world.